### ISSUE TITLE
Dropout: A Simple Way to Prevent Neural Networks from Overfitting

### ISSUE THREAD

**Issue by oba18** [oba18/survey-paper#6]
# 論文まとめ

## 0. 論文
Dropout: A Simple Way to Prevent Neural Networks from
Overfitting

## 1. どんなもの？
dropoutという過学習を防ぐためのシンプルな方法の提案．

## 3. 技術や手法のキモはどこ？
![image](https://user-images.githubusercontent.com/33748835/78954039-d073c880-7b15-11ea-838d-a09fc57b0a16.png)

![image](https://user-images.githubusercontent.com/33748835/78953570-3d865e80-7b14-11ea-8ac2-a9f9ca834128.png)

## 4. どうやって有効だと検証した？
大量のデータセットの種類で試した．
